content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -15,6 +15,14 @@ This page shows how to install the `kubeadm` toolbox.
 For information on how to create a cluster with kubeadm once you have performed this installation process,
 see the [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) page.
 
+This installation guide is for Kubernetes {{< skew currentVersion >}}. If you want to use a different
+Kubernetes version, please refer to the following pages instead:
+
+- [Installing kubeadm for Kubernetes {{< skew currentVersionAddMinor -1 "." >}}](https://v{{< skew currentVersionAddMinor -1 "-" >}}.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)
+- [Installing kubeadm for Kubernetes {{< skew currentVersionAddMinor -2 "." >}}](https://v{{< skew currentVersionAddMinor -2 "-" >}}.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)
+- [Installing kubeadm for Kubernetes {{< skew currentVersionAddMinor -3 "." >}}](https://v{{< skew currentVersionAddMinor -3 "-" >}}.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)
+- [Installing kubeadm for Kubernetes {{< skew currentVersionAddMinor -4 "." >}}](https://v{{< skew currentVersionAddMinor -4 "-" >}}.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)
```

Comment:
Instead of using this list we can add a similar note similar to this one.

https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

This can be added before the installation steps.

Reply:
I copied this code from [the Upgrading kubeadm clusters page](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/) but I see that it's also affected by the issue that you mentioned. Let's leave it as it is for now and I can create a follow-up issue once this PR is merged.

Reply:
> Instead of using this list we can add a note similar to this one.
> 
> https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

We could, but let's not.

We want to help readers find the right docs for their version (we _don't_ want to set an expectation that the Kubernetes website covers older versions). Let's avoid trapping ourselves into an expectationof backwards compatibility.